static/app/views/alerts/incidentRules/thresholdTypeForm.tsx

```diff
@@ -0,0 +1,82 @@
+import {Fragment, PureComponent} from 'react';
+import styled from '@emotion/styled';
+
+import Feature from 'sentry/components/acl/feature';
+import RadioGroup from 'sentry/components/forms/controls/radioGroup';
+import ListItem from 'sentry/components/list/listItem';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Organization} from 'sentry/types';
+
+import {AlertRuleComparisonType, Dataset} from './types';
+
+type Props = {
+  comparisonType: AlertRuleComparisonType;
+  dataset: Dataset;
+  disabled: boolean;
+  hasAlertWizardV3: boolean;
+  onComparisonTypeChange: (value: AlertRuleComparisonType) => void;
+  organization: Organization;
+};
+
+class ThresholdTypeForm extends PureComponent<Props> {
```

Comment:
lets just use a functional component, i don't think the form is that hard to re-render